dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -305,9 +306,15 @@ private static FieldSymbolReference constantToFieldReference(
     return parameterTypes;
   }
 
-  /** Returns the jar file URL of a class in the class path. */
+  /**
+   * Returns the jar file URL of a class in the class path. Null if the information is unavailable.
+   */
   URL findClassLocation(String className) throws ClassNotFoundException {
     Class<?> clazz = loadClass(className);
+    if (clazz.getProtectionDomain().getCodeSource() == null) {
```

Comment:
pull clazz.getProtectionDomain().getCodeSource() out to a local variable since you use it twice